src/sql/workbench/common/actions.ts

```diff
@@ -239,16 +239,23 @@ export class BackupAction extends Task {
 	}
 
 	runTask(accessor: ServicesAccessor, profile: IConnectionProfile): void | Promise<void> {
+		let configurationService = accessor.get<IConfigurationService>(IConfigurationService);
```

Comment:
nit: But you should try to use const where you can. These should all be able to be const.

Reply:
not new code, just moved :) will change